rotkehlchen/chain/ethereum/manager.py

```diff
@@ -393,55 +369,56 @@ def attempt_connect(
                         synchronized, msg = _is_synchronized(current_block, latest_block)
             except ValueError as e:
                 message = (
-                    f'Failed to connect to ethereum node {name} at endpoint '
+                    f'Failed to connect to ethereum node {node} at endpoint '
                     f'{ethrpc_endpoint} due to {str(e)}'
                 )
                 return False, message
 
             if not synchronized:
                 self.msg_aggregator.add_warning(
-                    f'We could not verify that ethereum node {name} is '
+                    f'We could not verify that ethereum node {node} is '
                     'synchronized with the ethereum mainnet. Balances and other queries '
                     'may be incorrect.',
                 )
 
-            log.info(f'Connected ethereum node {name} at {ethrpc_endpoint}')
-            self.web3_mapping[name] = web3
+            log.info(f'Connected ethereum node {node} at {ethrpc_endpoint}')
+            self.web3_mapping[node] = web3
+            self.own_rpc_endpoints.add(node)
```

Comment:
What is this Set for and why is it called `own_rpc_endpoint`?

Reply:
It was here in the previous logic for having all the own nodes but is not needed anymore since we have now a owned column